hashes/src/sha256t.rs

```diff
@@ -71,7 +71,6 @@ where
     pub fn from_engine(e: HashEngine) -> Hash<T> { from_engine(e) }
 
     /// Copies a byte slice into a hash object.
-    #[deprecated(since = "TBD", note = "Use `from_byte_array` instead.")]
```

Comment:
We should deprecate the other uses instead of un-deprecating this one.

Reply:
FTR, this is done.